frozen_requirements.txt

```diff
@@ -1,14 +1,8 @@
-# Copyright (c) 2020 Shotgun Software Inc.
-#
-# CONFIDENTIAL AND PROPRIETARY
-#
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
-# Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
-# not expressly granted therein are reserved by Shotgun Software Inc.
-
-requests==2.11.1
+certifi==2020.6.20
```

Comment:
Frozen requirements used to be the actual list of packages we install, since there wasn't any dependency, but now that file is generated by pip and requirements.txt is the actual list of stuff we want to install.